cluster-operator/src/main/java/io/strimzi/operator/cluster/model/RestartReason.java

```diff
@@ -28,11 +28,6 @@ public enum RestartReason {
      */
     CA_CERT_RENEWED("CA certificate renewed"),
 
-    /**
-     * Clients CA private key was replaced
-     */
-    CLIENT_CA_CERT_KEY_REPLACED("Trust new clients CA certificate signed by new key"),
```

Comment:
I guess this should be removed from docs as well (search for `ClientCaCertKeyReplaced`).